fragments/host-update.sh

```diff
@@ -25,7 +25,7 @@ source /usr/local/share/openshift-on-openstack/common_functions.sh
 if [ -e /run/ostree-booted ]
 then
     # Update the OS tree
-    atomic host upgrade || notify_failure "failed to run 'atomic host upgrade'"
+   [ "$SYSTEM_UPDATE" = "True" ] && (atomic host upgrade || notify_failure "failed to run 'atomic host upgrade'")
```

Comment:
Good catch. One nit, because [ "$SYSTEM_UPDATE" = "True" ] is checked in both branches of "if-then-else", it would be better to move "[ "$SYSTEM_UPDATE" = "True" ]" in front of the if statement, something like:

[ "$SYSTEM_UPDATE" = "True" ] || exit 0

if [ -e /run/ostree-booted ]
then
    # Update the OS tree
   atomic host upgrade || notify_failure "failed to run 'atomic host upgrade'"
else
    # Update using traditional RPMs
    retry yum install -y deltarpm || notify_failure "could not install deltarpm"
    retry yum -y update || notify_failure "could not update RPMs"
fi